guides/source/configuring.md

```diff
@@ -613,6 +613,17 @@ There are a few configuration options available in Active Support:
 
 * `config.active_job.logger` accepts a logger conforming to the interface of Log4r or the default Ruby Logger class, which is then used to log information from Active Job. You can retrieve this logger by calling `logger` on either an Active Job class or an Active Job instance. Set to `nil` to disable logging.
 
+### Configuring Action Cable
+
+* `config.action_cable.url` accepts a string for the URL for where
+ you are hosting your Action Cable server. You would use this option
+if you are running Action Cable servers that are separated from your
+main application.
+* `config.action_cable.mount_path` accepts a string for where to mount Action
+  Cable, as apart of the main server process. Defaults to `/cable`.
+You can set this as nil to not mount Action Cable as apart of your
+normal Rails server.
```

Comment:
I don't understand why these are two different settings? Why not just have `config.action_cable.url` and `config.action_cable.mounted = true/false`? It's pretty awkward to do `config.action_cable.mount_path = nil` to say you don't want it mounted.